packages/renderer-preact/src/__tests__/index.js

```diff
@@ -1,9 +1,10 @@
 /** @jsx h */
 
 import { Component, h } from 'preact';
-import withRenderer from '..';
+import { withRenderer } from 'skatejs';
+import withPreact from '..';
 
-class MyElement extends withRenderer() {
+class MyElement extends withRenderer(withPreact()) {
```

Comment:
I don't mind using `withRenderer` here in the tests. It simplifies it a bit to do so. Another way is, like you said, to just define it on the class and return a value it can use.

Reply:
Why didn't this work before? Or did it? If you remove `withRenderer`, and the accessing of `renderRoot`, everything works, right?

Reply:
You mean, just use the original code where you store `root` on `this._renderRoot`? I only updated it for consistency with the React renderer really, but I'm happy to revert.

Reply:
Interestingly, the current tests still pass with the accessing of `renderRoot`, would you rather I reverted these last changes to the Preact renderer?

Reply:
I've reverted the `renderRoot` changes to `renderer-preact`